acct-enricher-job/pom.xml

```diff
@@ -169,14 +170,13 @@
         <dependency>
             <groupId>org.flywaydb</groupId>
             <artifactId>flyway-core</artifactId>
-            <version>7.10.0</version>
+            <version>8.5.12</version>
```

Comment:
We can't update it yet. It is incompatible with the current version we have on the servers. But we can try a see if devops have updated it by now.